subsys/bluetooth/services/ras/rreq/ras_rreq.c

```diff
@@ -1207,9 +1207,17 @@ void bt_ras_rreq_rd_subevent_data_parse(struct net_buf_simple *peer_ranging_data
 					bt_ras_rreq_subevent_header_cb_t subevent_header_cb,
 					bt_ras_rreq_step_data_cb_t step_data_cb, void *user_data)
 {
-	if (!peer_ranging_data_buf || !local_step_data_buf || peer_ranging_data_buf->len == 0 ||
-	    local_step_data_buf->len == 0) {
-		LOG_ERR("Tried to parse empty step data.");
+	if (!peer_ranging_data_buf) {
+		LOG_ERR("No peer step data provided.");
+		return;
+	} else if (!local_step_data_buf) {
+		LOG_ERR("No local step data provided.");
+		return;
+	} else if (peer_ranging_data_buf->len == 0) {
+		LOG_ERR("Tried to parse empty peer step data.");
+		return;
+	} else if (local_step_data_buf->len == 0) {
+		LOG_ERR("Tried to parse empty local step data.");
 		return;
 	}
```

Comment:
```suggestion
	if (!peer_ranging_data_buf) {
		LOG_ERR("No peer step data provided.");
	} else if (peer_ranging_data_buf->len == 0) {
		LOG_ERR("Tried to parse empty peer step data.");
	}
	if (!local_step_data_buf) {
		LOG_ERR("No local step data provided.");
	} else if (local_step_data_buf->len == 0) {
		LOG_ERR("Tried to parse empty local step data.");
	}
	return;
```
minor suggestion to print two error messages instead of one if two cases hit